src/Constants.ts

```diff
@@ -43,3 +43,109 @@ export const PATTERN_POSITION_TABLE = [
 export const G15 = (1 << 10) | (1 << 8) | (1 << 5) | (1 << 4) | (1 << 2) | (1 << 1) | (1 << 0);
 export const G18 = (1 << 12) | (1 << 11) | (1 << 10) | (1 << 9) | (1 << 8) | (1 << 5) | (1 << 2) | (1 << 0);
 export const G15_MASK = (1 << 14) | (1 << 12) | (1 << 10) | (1 << 4) | (1 << 1);
+
+export interface LogoSize {
+    maxLogoSize: number;
+    maxLogoWidth: number;
+    maxLogoHeight: number;
+}
+
+// ER L
```

Comment:
Please update the comments, instead of `ER L` , please add a small comment mentioning what these values are . Same for other error correction levels